src/index.css

```diff
@@ -18,3 +20,46 @@ a {
   justify-content: center;
   text-decoration: none;
 }
+
+img {
+  width: 500px;
+}
+
+/* NewsCard */
+.card {
+  color: black;
+  background-position: 0 -60px !important;
```

Comment:
Try to minimize `!important` use as much as possible. You can add precedence to a css rule by making it longer, such as `div > .class-name .child-class`. If you can't easily make a standard rule, then `!important` would be ok.